imports/plugins/core/orders/client/templates/orders.js

```diff
@@ -141,6 +138,11 @@ Template.orders.helpers({
     return Template.instance().state.get("orders") || false;
   },
 
+  hasMoreOrders() {
+    const filter = Reaction.getUserPreferences(PACKAGE_NAME, ORDER_LIST_FILTERS_PREFERENCE_NAME, DEFAULT_FILTER_NAME);
+    return Template.instance().state.get("orders").length >= Template.instance().orderLimits.get(filter);
```

Comment:
The length of the orders in state is always 2 or less coz you set limit to 2 orders.  So it returns true everytime you have two or more orders. So if I actually have just 2orders in total it'll still show the "Show more" button. Which isn't ideal.